pkg/cloudprovider/providers/gce/token_source.go

```diff
@@ -86,23 +88,28 @@ func (a *altTokenSource) token() (*oauth2.Token, error) {
 		return nil, err
 	}
 	var tok struct {
-		AccessToken       string `json:"accessToken"`
-		ExpiryTimeSeconds int64  `json:"expiryTimeSeconds,string"`
+		AccessToken string `json:"accessToken"`
+		ExpireTime  string `json:"expireTime"`
```

Comment:
I think this will work if you define ExpireTime as a time.Time, and let the JSON decoder parse the time implicitly:
http://play.golang.org/p/RqBu8zxfTV